ivadomed/testing.py

```diff
@@ -217,11 +220,13 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
 
                     output_nii_shape = output_nii.get_fdata().shape
                     if len(output_nii_shape) == 4 and output_nii_shape[-1] > 1 and ofolder:
-                        imed_visualize.save_color_labels(np.stack(pred_tmp_lst, -1),
-                                                     False,
-                                                     fname_tmp,
-                                                     fname_pred.split(".nii.gz")[0] + '_color.nii.gz',
-                                                     imed_utils.AXIS_DCT[testing_params['slice_axis']])
+                        logger.warning('No color labels saved due to a temporary issue. For more details see:'
+                                       'https://github.com/ivadomed/ivadomed/issues/720')
+                        # imed_visualize.save_color_labels(np.stack(pred_tmp_lst, -1),
+                        #                              False,
+                        #                              fname_tmp,
```

Comment:
Is there a reason we are leaving this commented out code here?

Reply:
justified in eca1c5f089e1a3f5a5fefe248a7746af10ef7fff